addon/i3dio/node_classes/shape.py

```diff
@@ -533,7 +534,7 @@ def populate_xml_element(self):
             xml_i3d.SubElement(self.xml_elements['subsets'], 'Subset', subset.as_dict())
 
     def _process_bounding_volume(self):
-        bounding_volume_object = self.evaluated_mesh.mesh.i3d_attributes.bounding_volume_object
+        bounding_volume_object = self.evaluated_mesh.source_object.data.i3d_attributes.bounding_volume_object

```

Comment:
```suggestion
        bounding_volume_object = self.evaluated_mesh.mesh.original.i3d_attributes.bounding_volume_object
```
Every data type inheriting from `ID` should have a `original` attribute on them that refers back to the original datablock so perhaps that could be used? I don't know if the mesh itself has it as it is generated, but the evaluated object itself should definitely have it.

Reply:
Seems like a rather useless thing then if they don't actually maintain the original unless you have to follow the chain of originals all the way back? 😂

Reply:
yea, not really sure why its like that... Guess we will never find out 😂